### DISCUSSION TITLE
NILE : Faithfulness as a metric

### DISCUSSION THREAD

**Discussion by hyunssong** [eduardoblanco/csc696H-Spring24#22] [Q&A]
Are the definitions for fatifhfullness and robustness identical? Or is there any difference? Do you think these parameters are the right choice for the problem? What else can be measured?

**Comment by mayanks43** [comment]
Faithfulness and robustness aren't identical. Faithfulness is whether the explanation the model outputted is what the model is actually reasoning through its learned weights/activations. Robustness is a way to test faithfulness and many other properties where we check whether the model is resistant to small changes in input. The paper's goal is to test faithfulness and they use the concepts of sufficiency and comprehensiveness to measure it. The authors use sensitivity/robustness to edits on the explanations to measure performance on sufficiency. I guess there are many more methods to measure faithfulness in the literature and authors chose sufficiency and comprehensiveness as that is popular in the literature? Unsure.
